frontend/packages/dev-console/integration-tests/support/pages/functions/installOperatorOnCluster.ts

```diff
@@ -57,6 +57,19 @@ const installIfNotInstalled = (operator: operators, tries: number = 4, polling:
   });
 };
 
+export const waitForCRDs = (operator: operators) => {
+  cy.log(`Verify the CRD's for the "${operator}"`);
+  operatorsPage.navigateToCustomResourceDefinitions();
+  cy.byTestID('name-filter-input')
+    .clear()
+    .type('Pipeline');
+  cy.get('tr[data-test-rows="resource-row"]', { timeout: 300000 }).should('have.length', 4);
+  cy.get('[data-test-id="TektonPipeline"]', { timeout: 80000 }).should('be.visible');
+  cy.get('[data-test-id="PipelineResource"]', { timeout: 80000 }).should('be.visible');
+  cy.get('[data-test-id="PipelineRun"]', { timeout: 80000 }).should('be.visible');
+  cy.get('[data-test-id="Pipeline"]', { timeout: 80000 }).should('be.visible');
```

Comment:
This part should be parameterized, otherwise the `operator` argument makes little sense

Reply:
make sense, updated the code